Header header
geometry_msgs/Pose pose
geometry_msgs/Twist velocity
geometry_msgs/Twist accel
---
Header header
humanoid_catching/FallPoint[] points
